lib/runPrettier.js

```diff
@@ -0,0 +1,61 @@
+const { spawn } = require('child_process');
+const path = require('path');
+
+const cwd = process.cwd();
+
+const filePattern = 'src/**/*.js';
+
+function runPrettier({ write, listDifferent, config }) {
+  const prettierConfig = config || {};
+
+  const prettierPath = path.join(
+    require.resolve('prettier'),
```

Comment:
Maybe you could resolve relative to `package.json`, and join it with the value under `bin.prettier`: https://github.com/prettier/prettier/blob/9290dd88038a8c155730beac3201e2f861697503/package.json#L6